hub/util/tests/test_from_tfds.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import pytest
+
+from hub.util.check_installation import tfds_installed  # type: ignore
+
+requires_tfds = pytest.mark.skipif(
+    not tfds_installed(), reason="requires tensorflow_datasets to be installed"
+)
+
+
+@requires_tfds
+def test_from_tfds_to_path(local_storage):
+    from hub.util.from_tfds import from_tfds, from_tfds_to_path  # type: ignore
+    if local_storage is None:
+        pytest.skip()
+    hub_ds = from_tfds_to_path(
+        tfds_dataset_name="mnist",
+        split="test",
+        hub_ds_path=local_storage.root,
+        batch_size=100,
+    )
+    assert len(hub_ds) == 10000
+    assert hub_ds.image.shape.upper == (28, 28)
+    assert hub_ds.image.shape.lower == (28, 28)
+    assert hub_ds.image[1000].shape.upper == (28, 28)
+    assert hub_ds.image[1000].shape.lower == (28, 28)
```

Comment:
Would you mind computing a sample and asserting it's not all zero or all one? Just to make sure the data is actually getting loaded and it's not filling up with blanks